packages/client/src/runtime/core/engines/client/interpreter/QueryInterpreter.ts

```diff
@@ -27,7 +27,7 @@ export class QueryInterpreter {
     return this.interpretNode(queryPlan, this.#placeholderValues)
   }
 
-  private async interpretNode(node: QueryPlanNode, env: Env): Promise<Value> {
+  private async interpretNode(node: QueryPlanNode, env: ScopeBindings): Promise<Value> {
```

Comment:
Let's rename the parameter as well:
```suggestion
  private async interpretNode(node: QueryPlanNode, bindings: ScopeBindings): Promise<Value> {
```